test/instrumentation/express-utils.test.js

```diff
@@ -26,6 +26,17 @@ test('#getPathFromRequest', function (t) {
     t.equals(path, '/foo/bar');
     t.end();
   });
+
+  t.test('should return path for an invalid host header', function (t) {
+    const req = createRequest(
+      'https://test.com/foo/bar?query=value#hash',
+      // eslint-disable-next-line prettier/prettier, no-useless-escape
+      'invalid\host\name',
```

Comment:
Does that actually throw in the try/catch?

```
> new URL('http://invalid\\host\\name')
URL {
  href: 'http://invalid/host/name',
  origin: 'http://invalid',
  protocol: 'http:',
  username: '',
  password: '',
  host: 'invalid',
  hostname: 'invalid',
  port: '',
  pathname: '/host/name',
  search: '',
  searchParams: URLSearchParams {},
  hash: ''
}
> new URL('http://invalid\host\name')
URL {
  href: 'http://invalidhostame/',
  origin: 'http://invalidhostame',
  protocol: 'http:',
  username: '',
  password: '',
  host: 'invalidhostame',
  hostname: 'invalidhostame',
  port: '',
  pathname: '/',
  search: '',
  searchParams: URLSearchParams {},
  hash: ''
}
```

This works, er fails :)

```
> new URL('http://bad[hostname')
Uncaught TypeError [ERR_INVALID_URL]: Invalid URL
    at __node_internal_captureLargerStackTrace (node:internal/errors:496:5)
    at new NodeError (node:internal/errors:405:5)
    at new URL (node:internal/url:637:13) {
  input: 'http://bad[hostname',
  code: 'ERR_INVALID_URL'
}
```

Reply:
> Does that actually throw in the try/catch?

I think the URL contained the char `[` when I tried in the REPL. My bad